src/status_im/contexts/communities/events.cljs

```diff
@@ -229,10 +229,9 @@
     {:db (update-in db
                     [:communities community-id]
                     (fn [community]
-                      (-> community
```

Comment:
leftover from https://github.com/status-im/status-mobile/pull/18843#discussion_r1492227751

Reply:
reverted, addressed in https://github.com/status-im/status-mobile/pull/18870